src/main/java/org/elasticsearch/index/mapper/core/AbstractFieldMapper.java

```diff
@@ -360,6 +368,14 @@ protected String defaultPostingFormat() {
     protected String defaultDocValuesFormat() {
         return null;
     }
+    
+    protected boolean defaultDocValues() {
```

Comment:
I didn't want to make it abstract b/c the vast majority of mappers want the behavior here from the abstract mapper? But I'm fine adding it to completion field mapper explicitly.